src/TMxpSendTagHandler.cpp

```diff
@@ -32,30 +33,46 @@ TMxpTagHandlerResult TMxpSendTagHandler::handleStartTag(TMxpContext& ctx, TMxpCl
     if (href.contains(TAG_CONTENT_PLACEHOLDER, Qt::CaseInsensitive) || hint.contains(TAG_CONTENT_PLACEHOLDER, Qt::CaseInsensitive)) {
         mIsHrefInContent = true;
     }
+    // Entities in href and hint may contain | seperators, but there are
+    // no | chars in the entity names (allowed, at least), so interpolate
+    // first:
+    href = ctx.getEntityResolver().interpolate(href);
+    if (!hint.isEmpty())
```

Comment:
Here and anywhere else, the consistent code style in Mudlet is braces around ifs